GradientLoadingBar/Classes/Views/GradientActivityIndicatorView.swift

```diff
@@ -91,7 +91,7 @@ open class GradientActivityIndicatorView: UIView {
 
     // MARK: - Private methods
 
-    private func commonInit() {
+    func commonInit() {
```

Comment:
I'd prefer keeping this method `private`. It should only be called from within this class (once!) from the initializer, as otherwise the result will be undefined. Instead you can create a separate `commonInit()` in your subclass and call it from an overridden initializer:

```swift

override public init(frame: CGRect) {
    super.init(frame: frame)

    commonInit()
}

public required init?(coder: NSCoder) {
    super.init(coder: coder)

    commonInit()
}

private func commonInit() {
    // ...
}
```